cpp/src/gandiva/precompiled/types.h

```diff
@@ -92,14 +92,14 @@ int64 timestampaddMonth_int64_timestamp(int64, timestamp);
 int64 timestampaddQuarter_int64_timestamp(int64, timestamp);
 int64 timestampaddYear_int64_timestamp(int64, timestamp);
 
-int64 date_add_int32_timestamp(int32, timestamp);
-int64 add_int64_timestamp(int64, timestamp);
+int64 date_add_timestamp_int32(timestamp, int32);
+int64 add_timestamp_int64(timestamp, int64);
```

Comment:
Why did these get renamed/reordered? Your changes seem to touch only `/(date_sub|date_diff|subtract|timestampadd).*/`

Reply:
see above comment regarding reverting a previous commit